crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -213,6 +213,7 @@ fn apply_pbr_lighting(
         let light_contrib = lighting::point_light(in.world_position.xyz, light_id, roughness, NdotV, in.N, in.V, R, F0, f_ab, diffuse_color);
         direct_light += light_contrib * shadow;
 
+#ifdef STANDARD_MATERIAL_DIFFUSE_TRANSMISSION
         if diffuse_transmission > 0.0 {
```

Comment:
the shader def being present should mean it's > 0.0, could we remove the if from the shader? (same for the others)

Reply:
Hmm, we could, but with the use of `diffuse_transmission_texture` and `specular_transmission_texture` it's possible for the per-fragment value of `diffuse_transmission` and `specular_transmission` to be 0, even when the shader def is present. In those cases without the if we'd do a bunch or computations or texture reads that would be immediately discarded.

Reply:
I'm not sure how much that really matters. I suspect the overhead of the if statement would be worse than any potential extra work on average. It's not a big deal either way, but it's probably worth removing still.